pages/partners/index.tsx

```diff
@@ -46,6 +46,24 @@ const partnersData: PartnersListingProps[] = [
     imageHeight: '362px',
     url: 'https://www.lgbtqseniorhousing.org/',
   },
+  {
+    name: 'The Equality Fund',
+    imageSrc: '/ef.jpg',
+    imageHeight: '58px',
+    url: 'https://www.tbf.org/nonprofits/grant-making-initiatives/equality-fund',
+  },
+  {
+    name: 'LGBTQIA+ Aging Project',
+    imageSrc: '/lap.png',
```

Comment:
what do loading times look for these images? should we host these on s3 to reduce time for meaningful content?

Reply:
yea it's a lil slow, i'm p sure we also talked about s3 before. Idk what the decision was but that's out of scope of this ticket